desktopcollector/main.go

```diff
@@ -84,9 +85,17 @@ func newCommand(set otelcol.CollectorSettings) *cobra.Command {
 
 	rootCmd.Flags().IntVar(&httpPortFlag, "http", 4318, "The port number on which we listen for OTLP http payloads")
 	rootCmd.Flags().IntVar(&grpcPortFlag, "grpc", 4317, "The port number on which we listen for OTLP grpc payloads")
-	rootCmd.Flags().IntVar(&browserPortFlag, "browser", 8000, "The port number where we expose our data")
+	rootCmd.Flags().BoolVar(&openBrowserFlag, "openBrowser", true, "Open the browser automatically on launch")
+	rootCmd.Flags().IntVar(&browserPortFlag, "browserPort", 8000, "The port number where we expose our data")
```

Comment:
I think `kebab-case` is more traditional for CLI arguments

Reply:
![a65dca8d-7c4c-4abc-8b2e-eef24d018e7b-1738179709045](https://github.com/user-attachments/assets/5face0ab-b4ef-4d2a-b6a0-34147f8ba725)
kebab-it-is-then